tsdb_k8s_queries/README.md

```diff
@@ -0,0 +1,91 @@
+## TSDB k8s query track
+
+The main goal of the TSDB k8s query track is to measure the performance of common k8s integration search requests.
+The queries, index templates and corpus data try to match production as close as possible.
+
+### The search requests
+
+The search requests of the following visualizations have been included in this track:
+* Average CPU usage per pod.
+* Average memory usage per pod.
+* Average cpu per container.
+* Average memory per container.
+* Average node cpu usage per container.
+* Average node memory usage per container.
+* Unique deployment count.
+* Percentile cpu usage per container.
+* Status per pod.
+* Transmitted network usage per pod.
+
+The search request is sourced from Kibana Lens in the k8s integration. For each of this search request a template is defined in operations.
+From each template, three search requests are generated. First with the last 15 minutes filter, then the last 2 hours filter, and finally with a last 24-hour interval. In case a `date_histogram` is used in the query template then each of these variations uses a different interval. A fixed interval of 30 seconds, 1 minute, and 30 minutes respectively.
+
+The k8s visualizations that run these queries don't run very often or under a high query load.
+Often these visualizations are loaded and then sometime later re-loaded. This triggers the shards of the k8s pod and container data streams to go search-idle. However, indexing always happens in the background. When shards become search-active again, a refresh needs to occur as part of the search request. This track is designed to emulate this runtime behaviour.
+
+This is done by concurrently indexing while the searches are ran. By lowering the `index.search.idle.after` setting from 30s (which is the default) to 1s. And force fully reducing the query load, so that one search gets executed every 3 seconds.
```

Comment:
These indexing operations are the ones prefixed as `touch-` right (referenced from https://github.com/elastic/rally-tracks/pull/373/files#diff-18fad8cb6e0990bf9839cad4594f45f8cfde9d2a0a8cce9495614f74c5ab670cR1070-R1085)

For example I can see the following during an ongoing run

```
cpu_usage_per_pod_15_minutes,touch-pod-1-15_minutes
touch-pod-1-2_hours,cpu_usage_per_pod_2_hours
touch-pod-1-24_hours,cpu_usage_per_pod_24_hours
touch-pod-2-15_minutes,memory_usage_per_pod_15_minutes
memory_usage_per_pod_2_hours,touch-pod-2-2_hours
```

showing that `touch-...` is run in parallel.

I think we could clarify a bit more that these touch tasks do; they are special in the sense that `bulk-size` is defined to `1`.

We could also reword this paragraph; for example reading

> This is done by concurrently indexing while the searches are ran. By lowering the `index.search.idle.after` setting from 30s (which is the default) to 1s. And force fully reducing the query load, so that one search gets executed every 3 seconds.

I am not sure what is exactly meant here (maybe full stops should be removed?)

Reply:
@dliappis I've updated the readme. Is that paragraph now clearer?

Reply:
@martijnvg it looks great, thank you.